src/plays/service/PlaysService.ts

```diff
@@ -16,13 +17,22 @@ export default class PlaysService {
     private readonly stepsService: StepsService
   ) { }
 
-  public get(playId: string): Promise<Play> {
+  public get(playId?: string): Promise<Play | Plays[]> {
+    if (typeof(playId) === 'undefined') {
+      const deferred = Q.defer<Plays[]>();
+      const requestUrl = `/${Version.V2}${Resource.PLAYS}`;
+      this.client.get(requestUrl, {}, (error, response, body) => {
+        this.mapPlayPromises(requestUrl, deferred, error, response, body, EXPECTED_GET_STATUS);
+      });
+      return deferred.promise as any;
+    }
     const deferred = Q.defer<Play>();
     const requestUrl = `/${Version.V1}${Resource.PLAYS}/${playId}`;
```

Comment:
Only `requestUrl` is different here so that can be easily done like this:
```
const deferred = Q.defer<Play | Plays[]>();
const requestUrl = playId ? `/${Version.V1}${Resource.PLAYS}/${playId}` : `/${Version.V2}${Resource.PLAYS}`;
```